### PR TITLE
Problem: (Fix#205) Missing "Am i signing?" check

### PR DESCRIPTION
**Pull request by lezzokafka**
Solution:
- Added signature checking script;
- Aligned it with `thaler-testnet.md`;
- Added a quick way to check the block height, warning of full sync before node-join in `thaler-testnet.md `

### CODE REVIEW

**Review by landanhu — COMMENTED** [review]
Can we piggyback this PR to fix typo `buy` to `but` in the two related files? 🥰 

> Other platforms may work buy there is no guarantee.

**Comment by lezzokafka** [comment]
> Can we piggyback this PR to fix typo `buy` to `but` in the two related files? 🥰
> 
> > Other platforms may work buy there is no guarantee.

Sure! 😉

File: docs/getting-started/thaler-testnet.md [hunk 1]
```diff
@@ -330,6 +348,16 @@ You will be required to insert the following:
 
 Once the node-join transaction was successfully broadcasted, you should be able to see your Council node one the [testnet explorer](https://chain.crypto.com/explorer/council-nodes).
 
+To further check if the council node is signing blocks, kindly run this [scrip](https://github.com/crypto-com/chain-docs/tree/master/docs/getting-started/assets/signature_checking/check-validator-up.sh) with the flag ` --pubkey ` to specify the public key of your validator. For example:
```
Comment (tomtau) [inline docs/getting-started/thaler-testnet.md]:
```suggestion
To further check if the council node is signing blocks, kindly run this [script](https://github.com/crypto-com/chain-docs/tree/master/docs/getting-started/assets/signature_checking/check-validator-up.sh) with the flag ` --pubkey ` to specify the public key of your validator. For example:
```

Comment (lezzokafka) [reply docs/getting-started/thaler-testnet.md]: Thanks @tomtau 👍

**Review by tomtau — APPROVED** [review]

**Pull request merged**